rmgpy/molecule/graphTest.py

```diff
@@ -640,6 +640,64 @@ def test_getPolycyclicRings(self):
         for ring in expectedContinuousRings:
             self.assertTrue(ring in continuousRings)
 
+
+    def test_getLargestRing(self):
+        """
+        Test that the Graph.getPolycyclicRings() method returns only polycyclic rings.
+        """
+        vertices = [Vertex() for i in range(27)]
+        bonds = [
```

Comment:
Please add a comment of what this molecule name is or where it came from. Thanks